### ISSUE TITLE
options in question info.json is available in generate() but not in grade()

### ISSUE THREAD

**Issue by chen386** [PrairieLearn/PrairieLearn#1692]
Observed in python version of the server file, haven't tried in the js version.

**Comment by eliotwrobson** [comment]
@nwalters512 Is this still an issue? Looks like options should be available everywhere now: https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/question-servers/freeform.js#L506

**Comment by nwalters512** [comment]
Yep, I just tested this and logging out `data["options"]` in `grade(...)` shows the expected object.
